ui/components/app/assets/auto-detect-nft/auto-detect-nft-modal.tsx

```diff
@@ -96,13 +97,9 @@ function AutoDetectNftModal({ isOpen, onClose }: AutoDetectNftModalProps) {
           >
             <img src="/images/wallet-alpha.png" />
           </Box>
-          <Text
-            variant={TextVariant.bodyMd}
-            textAlign={TextAlign.Justify}
-            padding={0}
-          >
+          <Text variant={TextVariant.bodyMd} padding={0}>
```

Comment:
non-blocking: the Text component doesn't have any padding by default so I don't believe this is doing anything
```suggestion
          <Text variant={TextVariant.bodyMd}>
```